collaboration/code_review.md

```diff
@@ -0,0 +1,154 @@
+in the author checklist.
+
+You should then request one or more reviewers, unless the code is not yet ready for review. In that case, mark it as a
+draft PR and add `[WIP]` to the title so your colleagues know it's a work in progress.
+
+## The review process
+
+Code review is a conversation between the author of a PR and reviewer(s). Sometimes this conversation is short and
+simple; other times it is long and detailed. At the end of a successful code review, everyone involved will be
+comfortable merging the code in the PR to the main branch.
+
+### Automated reviewers
+
+Many repos have automations configured that check certain aspects of the code that are easily machine-readable.
+For example, Coveralls cnecks the percentage of the codebase that is covered by tests, and CodeClimate checks for style
+issues and cognitive complexity. At minimum, we recommend configuring a linter, so both author and reviewer are clear
+on what style conventions are expected.
+
+Both the author and the reviewer of the PR are responsible for considering these automated reviews. A drop in test
+coverage should prompt a conversation about whether additional tests are needed. Similarly, if CodeClimate flags a highly
+complex or repeated piece of code, the author and reviewer may want to discuss whether a refactor would be useful. On a
+more basic level, the author should fix any straightforward style issues raised by the linter (incorrect indentation,
+exceeding line width, etc.)
+
+### Role of the reviewer
+
+Fundamentally, the reviewer's job is to confirm that the code does what it is supposed to do. The reviewer also provides
+another perspective on how the feature is implemented. For example, they might suggest an approach that introduces
+less complexity, or identify where additional test coverage may be needed.
+
+A successful review requires care and mindfulness. Not only will you need to read the code closely, you will also need
+to be empathethic to the author. Here are a few things to consider as you review:
+
+* **Be compassionate.** If something is wrong with the code, or it doesn't work as expected, consider how to phrase this
+kindly. Remember that a lot of context is lost in the written word.
+* **Avoid nitpicking.** Suggestions to improve the code are welcome during the review process, but recommending minor
+changes that don't make a significant impact can unnecessarily extend the review. At worst, these nitpicks can strain
+the dynamic between the reviewer and author.
+* **Be specific.** When requesting a change, make sure to clarify what exactly needs to change and why. This can help divert
+any misunderstandings.
+* **Remember positional power.** If you are the author's supervisor, how does this affect the dynamic of the review?
+
+With these in mind, you can read through the code, making comments here appropriate. When you're finished, either
```

Comment:
Should this read `...comments where appropriate...` ?

Reply:
Fixed. Thanks for catching that!